cms/static/cms/js/modules/cms.base.js

```diff
@@ -324,6 +333,7 @@ CMS.API.Helpers = {
      * @param {String[]} [params] array of `param=value` strings to update the url
      * @returns {String}
      */
+    // TODO get rid of this
```

Comment:
this PR introducts URL.js for proper url handling, should use that instead of our own